ImageNet/inference/HuaweiCloud_resnet50_modelarts_2.4.json

```
@@ -0,0 +1,18 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "william.zhouweimin@huawei.com",
+    "framework": "TensorFlow 1.8.0",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/tree/master/DAWNBench_Inference",
+    "commitHash": "https://github.com/huawei-clouds/modelarts-example/commit/b8831d3e3e38ccbdbed7b7e8bc697622e02dc069",

```

Remove the rest of the URL

```suggestion
    "commitHash": "b8831d3e3e38ccbdbed7b7e8bc697622e02dc069",
```

---

Hi, commitHash has been updated. Please check again, thanks.